test/regression.uts

```diff
@@ -2587,6 +2587,29 @@ del(l[4])
 
++ Test read_routes6() function
+
+= Default output
+
+routes6 = read_routes6()
+# Expected results:
+# - one route if there is only the loopback interface
+# - three routes if there is a network interface
+
+iflist = get_if_list()
+if iflist == [scapy.arch.LOOPBACK_NAME]:
+    len(routes6) == 1
+elif len(iflist) >= 2:
+    len(routes6) >= 3
+
+= Check mandatory routes
+
+len(filter(lambda r: r[0] == "::1" and r[-1] == ["::1"], routes6)) >= 1
+if iflist >= 2:
+    len(filter(lambda r: r[0] == "fe80::" and r[1] == 64, routes6)) >= 1
```

Comment:
Because I think your test won't work. Try replacing `len(filter(lambda r: r[0] == "fe80::" and r[1] == 64, routes6)) >= 1` with `False`. Only the last statement of a test is checked.